share/spice/wikidata/wikidata.js

```diff
@@ -0,0 +1,25 @@
+(function (env) {
+    "use strict";
+
+    env.ddg_spice_wikidata = function(api_result){
+
+        if (!api_result || api_result.error) {
+            return Spice.failed('wikidata');
+        }
+
+        Spice.add({
+            id: "wikidata",
+            name: "Wikidata",
+            data: api_result.result.values,
+            meta: {
+                sourceName: "wikidata.org",
+                sourceUrl: 'http://www.wikidata.org/wiki/' + api_result.result.item
+            },
+            templates: {
+                group: 'media',
```

Comment:
I think the Info template is better:

```js
group: 'info'`
```

With this you shouldn't need to specify the detail template.